exampleSite/config.toml

```diff
@@ -70,9 +70,10 @@ paginate = 10
     defaultKeywords = ["devows", "hugo", "go"]
     defaultDescription = "Site template made by devcows using hugo"
 
-    # Twitter
-    twitter = "GoHugoIO"
-    twitterImage = "img/twitter-default.png"
+    # Social media
```

Comment:
the readme needs to be modified according these new parameter, also with [twitter/twitter]_author.

Reply:
ok

Reply:
I've updated (and enhanced) the respective section of the README, see 6fe633b127c4fe9fd8c137161e3046a6208cae61 and 24c72b7108fa7159c02fc379162212e7edb063a4